convert_hf_to_gguf.py

```diff
@@ -3167,6 +3167,24 @@ def set_gguf_parameters(self):
         self.gguf_writer.add_rope_scaling_type(gguf.RopeScalingType.NONE)
 
 
+@Model.register("Cohere2ForCausalLM")
+class Cohere2Model(Model):
+    model_arch = gguf.MODEL_ARCH.COHERE2
+
+    def set_gguf_parameters(self):
```

Comment:
The `config.json` has `"max_position_embeddings": 8192,` but the model supports 128K context. Do we need to adjust this value here?

Reply:
Don't quote me on this but I think it's fine to leave this as-is and force users to adjust rope settings to enable the full context